api/v1/controllers/AuthController.js

```diff
@@ -70,8 +80,85 @@ function refreshToken (req, res, next) {
 		});
 }
 
+function requestPasswordReset (req, res, next) {
+	var userEmail = req.query.email;
+
+	UserService
+		.findUserByEmail(userEmail)
+		.then(function (user){
+			return AuthService.generateToken(user, scopes.AUTH);
+		})
+		.then(function (tokenVal){
+			//TODO: Determine exact url route for reset page
+			tokenURL = (config.isDevelopment ? "localhost:8080/" : "hackillinois.org/") + "v1/auth/resetpage?token=" + tokenVal;
+			var substitutions = {'resetURL': tokenURL};
+			return MailService.send(userEmail, mail.templates.passwordReset, substitutions);
+		})
+		.then(function(){
+			res.body = {};
+			next();
+			return null;
+		})
+		.catch(function (error){
+			next(error);
+			return null;
+		});
+}
+
+function passwordReset(req, res, next) {
+	// MARK - Should be able to remove this and let middleware handle the requirements
```

Comment:
It looks like `endpoints.js` has not been modified, so the middleware doesn't know to enforce the password length requirement.